curated_tokenizers/_wordpiece.pyx

```diff
@@ -18,6 +18,19 @@ cdef class WordPieceProcessor:
             byte_array = piece[2:].encode('utf8') if not is_initial else piece.encode('utf8')
             self._pieces.add_piece(byte_array, is_initial)
 
+    def __copy__(self):
+        cls = self.__class__
+        data = self.to_list()
```

Comment:
Looks like a deepcopy, but there doesn't seem to be a nice other way to do this.